index.html

```diff
@@ -134,14 +134,14 @@ <h4 class="project-title">CC ignite</h4>
 
```

Comment:
Great design @maths29 🤝

- [ ] As stated in the projectrequiremets, kindly use the font family `Cocogoose, Lato` provided so that your design can replicate the design provided.